dist/dragula.js

```diff
@@ -86,6 +87,10 @@ function dragula (initialContainers, options) {
   if (o.removeOnSpill === true) {
     drake.on('over', spillOver).on('out', spillOut);
   }
+  
+  if(o.copySortSource === true) {
```

Comment:
we can change this to just `_copySortSource = o.copySortSource` (and remove the `_copySortSource` assignment from line 76)